tests/test_artifacts.py

```diff
@@ -0,0 +1,123 @@
+import os
```

Comment:
Do we check any failure case in this test?

Reply:
It's only used for removing the file.txt file

Reply:
I wasn't talking about the os import, I just wanted to add a comment to the whole file... But anyway, you told me that Artifactory cannot fail, so there is no test for that.
Maybe at least something like testing a download for which the artifact is not found?